keras/utils/visualize_util.py

```diff
@@ -23,17 +23,29 @@ def model_to_dot(model, show_shapes=False, show_layer_names=True):
         model = model.model
     layers = model.layers
 
-    # first, populate the nodes of the graph
+    # Create graph nodes.
     for layer in layers:
         layer_id = str(id(layer))
+        
+        # Append a wrapped layer's label to node's label, if it exists
+        # (e.g. TimeDistributed(Dense()) becomes "TimeDistributedDense",
+        # Bidirectional(LSTM()) becomes "BidirectionalLSTM", and so on).
```

Comment:
Ah, but I figured it'd be nice to follow the convention set by TimeDistributedDense and TimeDistributedMerge when those were still a thing. But I get your point, concatenating them makes it seem like there's actually a class called that. However, I figured that would be fine because the wrappers essentially follow a decorator pattern.

How about:
![image](https://cloud.githubusercontent.com/assets/1595907/19688889/c2e9dcaa-9acb-11e6-80ed-9ae10d79410c.png)

It would also be possible to use subgraphs (something in the style of plate notation for PGMs) for wrappers, but it requires some refactoring that I don't think is worth it.